libs/wingc/examples/compile.rs

```diff
@@ -12,8 +12,11 @@ pub fn main() {
 	}
 
 	let source_path = &args[1];
+	let source_path = Path::new(source_path);
 
-	let results = compile(Path::new(source_path), None);
+	std::env::set_var("WING_PROJECT_DIR", source_path.parent().unwrap());
```

Comment:
Not sure if I'm okay with this. Not having a way to canonicalize paths is really hurting.

The environment variable is currently only set in native test contexts.